perftests/network-ssl/test_network_ssl.sh

```diff
@@ -1,13 +1,32 @@
 #!/bin/sh
 
-set -e
+failed=0
+output=tests-stderr.log
 
+fail_test() {
+	hostname=$1
+
+	./test_network_ssl $hostname 2>> tests-stderr.log
+	ret=$?
+
+	if [ "$ret" -eq "0" ]; then
+		printf "Failed to fail on $hostname!\n"
+		failed=$((failed + 1))
+	fi
+}
+
+# Clear previous output
+rm -f "${output}"
+
+# We should pass this one
 ./test_network_ssl www.tarsnap.com
```

Comment:
Looks like you're not checking whether this succeeds now?

Reply:
Whoops.  Fixed now.